Dockerfile

```diff
@@ -1,16 +1,43 @@
-# To compile this image manually run:
-#
-# $ packr; GO111MODULE=on GOOS=linux GOARCH=amd64 go build; docker build -t oryd/keto:latest .; rm keto; packr clean
-FROM alpine:3.9
+FROM golang:1.16-alpine AS builder
```

Comment:
Please follow the structure of https://github.com/ory/kratos/tree/master/.docker

The main Dockerfile is an empty container where goreleaser injects its binary into.